evennia/objects/objects.py

```diff
@@ -2986,7 +3037,8 @@ def create_exit_cmdset(self, exidbobj):
     # Command hooks
 
     @classmethod
-    def create(cls, key, source, dest, account=None, **kwargs):
+    def create(cls, key: str, location: DefaultRoom = None, destination: DefaultRoom = None, account: "DefaultAccount" = None, creator: DefaultObject = None,
```

Comment:
Use `caller` instead of `creator` here.

Good call on changing source/dest to location/destination; that does read better. :+1:

Reply:
We have entirely new methods here, like `get_object_typeclass` that has no unit tests on it.